gsoc/gsoc2016/ideas_eras.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: default
+title:  "Ideas for IMS - ERAS"
+show_main: false
+ideas_team: IMS
+---
+
+### 1 – SOLAR STORMS FORECASTING SERVER ###
+
+*Suggested Mentors:* [Antonio del Mastro](https://bitbucket.org/aldebran/) , [Olena Persianova](https://bitbucket.org/OPersian/)
```

Comment:
I don't know what's your background but if you need someone with solar physics knowledge to help on this I can help with that.